src/jit/emitxarch.cpp

```diff
@@ -955,6 +955,13 @@ unsigned emitter::emitGetVexPrefixAdjustedSize(instruction ins, emitAttr attr, c
 
         return vexPrefixAdjustedSize;
     }
+
+    if (Is4ByteSSEInstruction(ins))
```

Comment:
This is here because it is more convenient to centralize it than it is to have explicit `Is4ByteSSEInstruction` checks at each required location.

Reply:
Fixed.